tests/import_test.go

```diff
@@ -1203,3 +1284,20 @@ var _ = Describe("Preallocation", func() {
 		Expect(ok).To(BeTrue())
 	})
 })
+
+func generateRegistryOnlySidecar() *unstructured.Unstructured {
```

Comment:
istio doesn't publish a k8s client?

Reply:
We only use a single trivial Istio resource (Sidecar). Do we really want to import 10k lines of code worth of dependencies to have the [istio structs and go-client](https://github.com/istio/client-go)?

Reply:
Hmm I figured it would only pull in the things we actually use, but if it is adding 10k lines of code just for something small, I can see why you went this route.